samples/sample_encode/include/pipeline_encode.h

```diff
@@ -29,6 +29,8 @@ or https://software.intel.com/en-us/media-client-solutions-support.
 
 #include "sample_utils.h"
 #include "base_allocator.h"
+
+#define TIME_STATS 1
```

Comment:
@egorovdanil , I don't understand what change. Can you please explain how this define helps?

Reply:
To output FPS, in the `pipeline_encode.cpp` it checks if a variable is defined:
`#ifdef TIME_STATS
        mfxF64 ProcDeltaTime = m_statOverall.GetDeltaTime() - m_statFile.GetDeltaTime() - m_TaskPool.GetFileStatistics().GetDeltaTime();
        msdk_printf(MSDK_STRING("Encoding fps: %.0f\n"), m_FileWriters.first->m_nProcessedFramesNum / ProcDeltaTime);
#endif`

Reply:
@AntonGrishin , I know you've recently fixed some issue with FPS reporting in samples. Was it a fix for sample_encode?

Reply:
@dmitryermilov I only enabled fps report in smt, but I didn`t touch sample encode.

Reply:
Okay. For it such conditional compilation doesn't makes  sense.  @egorovdanil , please move this fps reporting outside of #ifdef

Reply:
done